http/director/proxy.go

```diff
@@ -140,9 +140,6 @@ func AuthMiddleware(authorizer authorize.Authorizer) httpwares.Middleware {
 
 func respondWithUnauthorized(err error, req *http.Request, resp http.ResponseWriter) {
 	status := http.StatusUnauthorized
-	if rErr, ok := (err).(*router.Error); ok {
```

Comment:
Because it does not make sense. Nothing which calls `respondWithUnauthorized` is using anything from `router.` package